public/src/news.js

```diff
@@ -0,0 +1,59 @@
+const app = document.getElementById('root')
+const newlist = document.querySelector('.new-list')
+
+const logo = document.createElement('img')
+logo.src='images/logo.png'
+
+const container = document.createElement('div')
+container.setAttribute('class', 'container')
+
+
+app.appendChild(logo)
+app.appendChild(container)
+
+
+fetch('https://content.guardianapis.com/search?show-blocks=all&api-key=3d1bd118-3d6f-4211-a61e-3d0f966b08bb')
+
+.then(response => {
+  return response.json()
+  
+})
+.then(data => {
+    data.response.results.forEach(news => {
+     var li = document.createElement('li')
+     var a = document.createElement('a')
+     a.setAttribute('href', onclick=retrieve)
```

Comment:
What does this line do?

Reply:
<img width="481" alt="Ekran Resmi 2020-06-02 07 49 29" src="https://user-images.githubusercontent.com/61015092/83488956-98e62480-a4a5-11ea-8077-1596477e57b9.png">

i deleted li html element now. 
Creating a element in every each news and then when user clicks in the browser, it triggers retrieve function which runs summarized article.

Reply:
Problem here is, when i click in anywhere in the browser, it works.(the retrieve function works). It is supposed to work when i only click 'see article' writing.